tools/utils/repo.py

```diff
@@ -11,15 +10,19 @@
 
 import tools.utils
 
+import logging
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
 try:
     import boto3
-    from botocore.exceptions import ClientError
+    from botocore.exceptions import ClientError, NoCredentialsError, PartialCredentialsError
 except ImportError:
-    print(
-        "\nPlease run 'python -m pip install -r "
-        "requirements/static/ci/py{}.{}/tools.txt'\n".format(*sys.version_info),
-        file=sys.stderr,
-        flush=True,
+    logger.error(
+        "Please run 'python -m pip install -r requirements/static/ci/py{}.{}tools.txt'".format(
+            *sys.version_info
+        )
```

Comment:
I suggest using the the formatting capability of logger instead of calling `format` explicitly. Something like
```
log.error("Please run 'python -m pip install -r requirements/static/ci/py%s.%stools.txt'", *sys.version_info)
```